src/main/java/org/squiddev/cobalt/LuaError.java

```diff
@@ -76,6 +76,7 @@ public LuaError(Throwable cause) {
 		level = 1;
 		calculateLevel = true;
 		value = ValueFactory.valueOf("vm error: " + cause.toString());
+		cause.printStackTrace(System.err);
```

Comment:
I assume this can be reverted?

Reply:
Yeah, I'm just leaving it in while debugging things.